src/cli/commands/color_commands.rs

```diff
@@ -148,6 +149,35 @@ color_command!(SetCommand, config, matches, color, {
                 "lab-b" => {
                     lab.b = value;
                 }
+                "luminance" => {
+                    let mut luminance = lab.into_color().luminance();
+
+                    let mut max_l = 100.0;
+                    let mut min_l = 0.0;
+                    let mut past_l = lab.l;
+
+                    // Binary search to get to the luminance value we want by adjusting the
+                    // lightness value in lab
+                    while (luminance - value).abs() > 0.001 {
+                        if luminance > value {
+                            max_l = lab.l;
+                        } else {
+                            min_l = lab.l;
+                        }
+
+                        lab.l = (max_l + min_l) / 2.0;
+
+                        // If we haven't made any changes, get out before we loop infinitely
+                        if past_l == lab.l {
```

Comment:
is this really the only thing that could happen? Couldn't the `L` value also toggle between two values? To be safe, we could also use an iteration counter.

Reply:
I agree, I'll put in an iteration counter.